aws-s3/renv.lock

```diff
@@ -2030,7 +2030,7 @@
     },
     "osmdata": {
       "Package": "osmdata",
-      "Version": "0.2.2",
+      "Version": "0.2.5",
```

Comment:
**issue:** The hash value for the package has almost certainly changed too. Usually `renv.lock` is updated with `renv::snapshot()`, which would also change the hash version.

Reply:
Good catch! I now know I should use renv always instead of manually editing the file.